indexer/project/hca/config.py

```diff
@@ -5,9 +5,9 @@
 from elasticsearch import Elasticsearch, RequestsHttpConnection
 from requests_aws4auth import AWS4Auth
 
-from utils.base_config import BaseIndexProperties
-from utils.deployment import aws
-from utils.transformer import Transformer
+from indexer.utils.base_config import BaseIndexProperties
```

Comment:
The `indexer` directory will go away soon. It should not be part of the package path. Set PYTHONPATH to point to `indexer` and undo these changes.